client/my-sites/themes/theme-showcase.jsx

```diff
@@ -35,9 +35,6 @@ import prependThemeFilterKeys from 'state/selectors/prepend-theme-filter-keys';
 import { recordTracksEvent } from 'state/analytics/actions';
 import ThemesSearchCard from './themes-magic-search-card';
 import QueryThemeFilters from 'components/data/query-theme-filters';
-import PhotoBlogBanner from './themes-banner/photo-blog';
-import SmallBusinessBanner from './themes-banner/small-business';
-import RandomThemesBanner from './themes-banner/random-themes-banner';
```

Comment:
Should we also delete these files now that they're no longer used or referenced anywhere?

Reply:
Good thinking. @crunnells did you want to take that on in a separate PR?

Reply:
Yah, can do!